### PR TITLE
Account Info UX Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#16838

*Description of changes:*
- Minor UX adjustments to AccountInfo Component

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/common/AccountInfo.vue [hunk 1]
```diff
@@ -109,6 +109,9 @@ export default defineComponent({
   .account-name,
   i {
     color: $gray9 !important;
+    span {
+      padding-left: 2px;
```
Comment (cameron-eyds) [inline ppr-ui/src/components/common/AccountInfo.vue:113]: Adjustment was too small for a helper class (4px increments)

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1479-axn33ssu.web.app

**Review by doug-lovett — APPROVED** [review]
Got to love the tiny adjustments.

**Review by orelbn — APPROVED** [review]
Nice

**Pull request merged**